.prettierignore

```diff
@@ -1 +1,2 @@
-.next/
\ No newline at end of file
+.next/
+*.md
```

Comment:
Why do we need to ignore .md files?

Reply:
Actually prettier is behaving very weird with .md files. It is changing almost half the content to some random character on save. So, I used different plugin to format .md files.

Reply:
ok okay 👍